commit 9a3f54a8b05c75efe24ad248cf33bbf1acd4835c (refs/changes/83/4483/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-07-30T12:26:23-07:00 (1 year, 2 months ago)
    
    TRITON-568 CNAPI should stop using restify.authorizationParser()
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/server.js b/lib/server.js
index aed3911..acd0644 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var os = require('os');
@@ -52,7 +52,6 @@ function createServer(options) {
     });
 
     cnapi.use(restify.acceptParser(cnapi.acceptable));
-    cnapi.use(restify.authorizationParser());
     cnapi.use(restify.queryParser({
         mapParams: true,
         allowDots: false,
diff --git a/package.json b/package.json
index 0411506..223f3e1 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.11.1",
+  "version": "1.11.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
